torch/distributed/nn/api/remote_module.py

```diff
@@ -279,7 +279,7 @@ def parameters(self, recurse: bool = True) -> Iterator[Parameter]:
 
     def named_parameters(  # type: ignore[return]
```

Comment:
shouldn't we remove this type ignore as well since we fixed the return Parameter

Reply:
```
torch/distributed/nn/api/remote_module.py:280: error: Missing return statement  [return]
```

Reply:
lol. Got it.